jwst/outlier_detection/outlier_detection.py

```diff
@@ -307,23 +298,33 @@ def blot_median(self, median_model):
         sinscl = self.outlierpars.get('sinscl', 1.0)
 
         # Initialize container for output blot images
-        blot_models = datamodels.ModelContainer()
+        blot_models = datamodels.ModelContainer(open_models=False)
 
         log.info("Blotting median...")
-
         for model in self.input_models:
-            blotted_median = model.copy()
+            blotted_median = model
             blot_root = '_'.join(model.meta.filename.replace(
                 '.fits', '').split('_')[:-1])
-            blotted_median.meta.filename = '{}_blot.fits'.format(blot_root)
+            model_path = self.make_output_path(
+                basename=blot_root,
+                suffix='blot'
+            )
+
+            blotted_median.meta.filename = model_path
 
             # clean out extra data not related to blot result
-            blotted_median.err = None
-            blotted_median.dq = None
+            blotted_median.err *= 0.0  # None
+            blotted_median.dq *= 0     # None
+
             # apply blot to re-create model.data from median image
             blotted_median.data = gwcs_blot(median_model, model, interp=interp,
                                             sinscl=sinscl)
-            blot_models.append(blotted_median)
+
+            blotted_median.save(model_path)
+            blot_models.append(model_path)
+            blotted_median.close()
+            # blot_models.append(blotted_median)
```

Comment:
Is this needed in the final (production) code?
```suggestion
```

Reply:
Removed in [43ea4f9](https://github.com/spacetelescope/jwst/pull/6904/commits/43ea4f99f01bf738e2818f3e1009a1ea56f30e3b)